native/acter/src/api/calendar_events.rs

```diff
@@ -222,6 +222,21 @@ impl CalendarEvent {
         crate::ReactionManager::new(client, room, event_id).await
     }
 
+    pub async fn participants(&self) -> Result<Vec<String>> {
+        let calendar_event = self.clone();
+        let mut user_ids = Vec::new();
+        RUNTIME
+            .spawn(async move {
+                let manager = calendar_event.rsvps().await?;
+                for u_id in manager.users_at_status("yes".to_string()).await? {
+                    let id_str = u_id.to_string();
+                    user_ids.push(id_str)
+                }
+                Ok(user_ids)
```

Comment:
can be simplified to:
```suggestion
        RUNTIME
            .spawn(async move {
                let manager = calendar_event.rsvps().await?;
                Ok(manager.users_at_status("yes".to_string()).await?.into_iter().map(|u_id| u_id.to_string()).collect())
```

Reply:
This throws me exception, not sure what's wrong here:
```
mismatched types
  expected enum `std::result::Result<Vec<std::string::String>, anyhow::Error>`
found unit type `()`rustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B0%5D?0#file%3A%2F%2F%2FUsers%2Fgtalhaa%2FWork%2Fflutter-projects%2Fa3%2Fnative%2Facter%2Fsrc%2Fapi%2Fcalendar_events.rs)
```